src/sentry/static/sentry/app/views/organizationEvents/events.jsx

```diff
@@ -0,0 +1,124 @@
+import {Link} from 'react-router';
+import React from 'react';
+import styled from 'react-emotion';
+
+import {Panel, PanelBody, PanelHeader} from 'app/components/panels';
+import AsyncView from 'app/views/asyncView';
+import DateTime from 'app/components/dateTime';
+import IdBadge from 'app/components/idBadge';
+import Pagination from 'app/components/pagination';
+import SentryTypes from 'app/sentryTypes';
+import Tooltip from 'app/components/tooltip';
+import withOrganization from 'app/utils/withOrganization';
+
+class OrganizationEvents extends AsyncView {
+  static propTypes = {
+    organization: SentryTypes.Organization,
+  };
+
+  constructor(props) {
+    super(props);
+    this.projectsMap = new Map(
+      props.organization.projects.map(project => [project.id, project])
+    );
+  }
+
+  getTitle() {
+    return `${this.props.organization.slug} Events`;
+  }
+
+  getEndpoints() {
+    return [['events', `/organizations/${this.props.organization.slug}/events/`]];
+  }
+
+  getEventTitle(event) {
+    const {organization} = this.props;
+    const project = organization.projects.find(({id}) => id === event.projectID);
+    return (
+      <Link to={`/${organization.slug}/${project.slug}/issues/?query=${event.eventID}`}>
+        {event.message.split('\n')[0].substr(0, 100)}
```

Comment:
Do you need to worry about potentially slicing 4 byte characters in half? substr() is not entirely unicode aware.

Reply:
Good catch, I don't think it's an issue to be concerned about since `100` is an arbitrary cut off anyway, and we're aiming to just give a summary.